test/e2e/lib/driver-manager.js

```diff
@@ -289,6 +289,13 @@ export async function ensureNotLoggedIn( driver ) {
 	await driver.executeScript(
 		'window.document.cookie = "sensitive_pixel_option=no;domain=.wordpress.com;SameSite=None;Secure"'
 	);
+
+	// JUST FOR TESTING
+	process.env.FLAGS = 'nav-unification';
+
+	process.env.FLAGS &&
+		( await driver.manage().addCookie( { name: 'flags', value: process.env.FLAGS } ) );
+
```

Comment:
Do I need to note this new flag in the e2e documentation? 

If so, would this description be correct:
`Controls whether nav-unification feature should be enabled. Set to `nav-unification` to turn on the feature.`

Reply:
I think we can skip it. It is temporary and currently only for nav unification. If we decide to use it for forcing others in the future it would be good to add. They are already at 75% on nav unification, so it will come out soon.

Reply:
I agree that we can skip it!